app/services/problem_check/starttls_disabled.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ProblemCheck::StarttlsDisabled < ProblemCheck
+  self.priority = "high"
+
+  def call
+    if GlobalSetting.smtp_enable_start_tls
+      no_problem
+    else
+      problem
+    end
```

Comment:
would we want to check for an explicit false here? this would cause a "problem" if it's nil/not set right?